rerun_js/web-viewer/index.ts

```diff
@@ -126,12 +143,13 @@ export class WebViewer {
       : undefined;
 
     this.#handle = new WebHandle_class({ ...options, fullscreen });
-    await this.#handle.start(this.#canvas.id);
-    if (this.#state !== "starting") return;
-
-    if (this.#handle.has_panicked()) {
-      throw new Error(`Web viewer crashed: ${this.#handle.panic_message()}`);
+    try {
+      await this.#handle.start(this.#canvas.id);
+    } catch (e) {
+      this.stop();
+      throw e;
     }
```

Comment:
Where did the `has_panicked` check go?

Reply:
Please make sure panics are still reported nicely, both during startup and runtime, otherwise we will have a hell of a time debugging this in the future

Reply:
looks about same as before. i just wanted to make sure `destroy` and `free` are called even in case we panic, to avoid leaks.

![image](https://github.com/rerun-io/rerun/assets/1665677/8534e91d-63ca-4860-ad2f-043e30defec1)